config/webpack.common.config.js

```diff
@@ -132,11 +132,10 @@ const commonWebpackConfig = {
         }, {
           from: RESOURCES_PATH + 'img/',
           to: './resources/img/'
+        }, {
+          from: RESOURCES_PATH + 'help/',
```

Comment:
It seems as if no such folder exists ...`src/resources/help`.

Reply:
right, has been added. thanks!